### ISSUE TITLE
pip install of rpy2 3.2.6 fails with python 3.5.5

### ISSUE THREAD

**Issue by madwsa** [rpy2/rpy2#654]
##### Traceback:

```
 Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/v6/5z5nf8rs0yv5w6tb5v0lvb6jcrqwpn/T/pip-install-uixpei1r/rpy2/setup.py", line 183, in <module>
        'rinterface_lib/R_API_eventloop.h']}
      File "/Users/a1452330/miniconda3/envs/obrp_scoe/lib/python3.5/site-packages/setuptools/__init__.py", line 140, in setup
        return distutils.core.setup(**attrs)

      File "/Users/a1452330/miniconda3/envs/obrp_scoe/lib/python3.5/distutils/core.py", line 108, in setup
        _setup_distribution = dist = klass(attrs)
      File "/Users/a1452330/miniconda3/envs/obrp_scoe/lib/python3.5/site-packages/setuptools/dist.py", line 370, in __init__
        k: v for k, v in attrs.items()
      File "/Users/a1452330/miniconda3/envs/obrp_scoe/lib/python3.5/distutils/dist.py", line 281, in __init__
        self.finalize_options()
      File "/Users/a1452330/miniconda3/envs/obrp_scoe/lib/python3.5/site-packages/setuptools/dist.py", line 529, in finalize_options
        ep.load()(self, ep.name, value)
      File "/private/var/folders/v6/5z5nf8rs0yv5w6tb5v0lvb6jcrqwpn/T/pip-install-uixpei1r/rpy2/.eggs/cffi-1.14.0-py3.5-macosx-10.9-x86_64.egg/cffi/setuptools_ext.py",
 line 217, in cffi_modules
        add_cffi_module(dist, cffi_module)
      File "/private/var/folders/v6/5z5nf8rs0yv5w6tb5v0lvb6jcrqwpn/T/pip-install-uixpei1r/rpy2/.eggs/cffi-1.14.0-py3.5-macosx-10.9-x86_64.egg/cffi/setuptools_ext.py",
 line 49, in add_cffi_module
        execfile(build_file_name, mod_vars)
      File "/private/var/folders/v6/5z5nf8rs0yv5w6tb5v0lvb6jcrqwpn/T/pip-install-uixpei1r/rpy2/.eggs/cffi-1.14.0-py3.5-macosx-10.9-x86_64.egg/cffi/setuptools_ext.py",
 line 24, in execfile
        code = compile(src, filename, 'exec')
      File "rpy2/_rinterface_cffi_build.py", line 127
        """,
          ^
    SyntaxError: invalid syntax
```

##### To reproduce:
Build a Conda environment from this environment.yaml:
```
channels:
  - defaults
  - conda-forge
dependencies:
  - python=3.5.5
  - pip:
    - rpy2
```

 . . . or just try `pip install rpy2` somewhere where 3.5.5 is the python version.

Looks like there is an f string in `rpy2/_rinterface_cffi_build.py`; this is supported in python 3.6 but not 3.5.5.

Editorializing here -- I know I can workaround by upgrading to python 3.6, but I can't do that where I am right now.  Please either require 3.6 or use `str.format()`.

**Comment by lgautier** [comment]
Thanks for the report. This is only in branch `v3.2.x`. I missed this.

**Comment by lgautier** [comment]
Fixed in revision 8d798da099947abdfd2d394c9a87cccc6ae3984b
